packs/pf2e/feats/ancestry/human/dragon-spit.json

```diff
@@ -37,7 +37,7 @@
                             "or": [
                                 "item:slug:acid-splash",
                                 "item:slug:electric-arc",
-                                "item:slug:produce-flame",
+                                "item:slug:ignition",
```

Comment:
Should we include `"item:slug:produce-flame"` as wel? Might be desired if using the pre-remaster content.